lib/src/universal_ble_web/universal_ble_web.dart

```diff
@@ -111,6 +111,13 @@ class UniversalBleWeb extends UniversalBlePlatform {
   Future<void> startScan({
     ScanFilter? scanFilter,
   }) async {
+    if (scanFilter == null || scanFilter.withServices.isEmpty) {
+      UniversalBlePlatform.logInfo(
+        "It is recommended to use ScanFilter.withServices on the web to ensure the specified services can be used after connecting",
```

Comment:
```suggestion
        "Error: services list empty. On web you have to specify services in the ScanFilter in order to be able to access those after connecting",
```
Can we log it as an error instead of info?